go.mod

```diff
@@ -1,6 +1,8 @@
 module istio.io/istio
 
-go 1.20
+go 1.21
+
+toolchain go1.21.1
```

Comment:
I think this is not needed

Reply:
I'll try to fix this so that the 1.0.0 can be included in the rc.0

Reply:
Thanks.